tests/test_dag/test_task2.py

```diff
@@ -0,0 +1,6 @@
+from dags.test_dag import task_2
+
+
+def test_task2():
+    value = task_2()
+    assert value == "second_task"
```

Comment:
Can we somehow test the DAG as well? https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#unit-tests